git-sv.hcl

```
@@ -0,0 +1,48 @@
+binaries = ["git-sv"]
+
+platform "darwin" "amd64" {
+  source = "https://github.com/thegeeklab/git-sv/releases/download/v${version}/git-sv-darwin-10.12-amd64"
```

It looks like these can all these platform blocks can be removed?
```suggestion
  source = "https://github.com/thegeeklab/git-sv/releases/download/v${version}/git-sv-${os}-10.12-${arch}"
```

---

This will give us linux/arm64 as well

---

good call. updated.